<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstQuestion</class>
 <widget class="QDialog" name="FirstQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>484</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>32</green>
        <blue>32</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>233</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>32</green>
        <blue>32</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>233</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>233</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>207</red>
        <green>233</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Do you live alone or do you have flatmates?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer1">
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Alone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer2">
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1-2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer3">
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3-4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer4">
        <property name="font">
         <font>
          <family>Gill Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Check">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
